litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -151,6 +156,7 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 	}
 
 	workflow_id := utils.RandomString(32)
```

Comment:
Why are we using RandomString for this? I think we should use uuid. It has a much less chance of colliding.

Reply:
Yes, make sense. Will update it

Reply:
Also we are not doing the project collection normalization?

Reply:
The project collection normalization PR will go in a separate PR.